bot/exts/events/advent_of_code/_cog.py

```diff
@@ -217,31 +217,31 @@ async def aoc_day_and_star_leaderboard(
         brief="Get a snapshot of the PyDis private AoC leaderboard",
     )
     @whitelist_override(channels=AOC_WHITELIST_RESTRICTED)
-    async def aoc_leaderboard(
-            self,
-            ctx: commands.Context,
-            self_placement_name: Optional[str] = None,
-    ) -> None:
+    async def aoc_leaderboard(self, ctx: commands.Context, *, aoc_name: Optional[str] = None) -> None:
         """
         Get the current top scorers of the Python Discord Leaderboard.
 
-        Additionally you can specify a `self_placement_name`
-        that will append the specified profile's personal stats to the top of the leaderboard
+        Additionally you can specify an `aoc_name` that will append the
+        specified profile's personal stats to the top of the leaderboard
         """
+        # Strip quotes from the self placement name if needed (e.g. "My Name" -> My Name)
+        if aoc_name and aoc_name.startswith('"') and aoc_name.endswith('"'):
```

Comment:
Could do:
```suggestion
        if all(aoc_name, aoc_name.startswith('"'), aoc_name.endswith('"')):
```

Reply:
This wouldn't work. `all()` take 1 argument, which needs to be an iterable. This means you need to put the 3 checks into a iterable, such as a tuple. This would mean all 3 get evaluated first, then sent to all, meaning if `aoc_name` is None, this would error, as it's no longer a lazy check.